### ISSUE TITLE
podman run: --tty should disable --sig-proxy

### ISSUE THREAD

**Issue by edsantiago** [projectatomic/libpod#983]
`podman-run(1)` documentation for `--sig-proxy` (copied from docker) states:
```
Proxy received signals to the process (non-TTY mode only).
```
However, signals are being proxied even with `--tty=on`.

In window 1:
```
# podman run --tty=true --sig-proxy=true fedora bash -c 'trap "echo got it" 10;while :;do sleep 0.1;done'
```
In window 2:
```
# ps auxww |grep podman|grep tty
root      1522  0.7  0.6 678500 24648 pts/3    Sl+  21:50   0:00 podman run --tty=true --sig-proxy=true fedora bash -c trap "echo got it" 10;while :;do sleep 0.1;done
# kill -10 1522
```
You will see 'got it' in window 1. (Expectation: no output in window 1)

podman-0.6.3-1.gitcd39b62.fc28 and .fc27

**Comment by mheon** [comment]
In this case, it is the docs that are wrong.

Ages ago, I worked a bug on Docker to disable this behavior there, but it never got merged. There is no technical reason for `--tty` to conflict with `--sig-proxy` here or there. Let's fix the docs and leave behavior unchanged.

**Comment by edsantiago** [comment]
WFM!

**Comment by mheon** [comment]
Fixed by #984
